objects/SetupChecker/script.lua

```diff
@@ -2083,6 +2107,14 @@ function pickGainSpiritChoice(player, guid, id)
     local aspect = obj.UI.getAttribute(id, "aspect")
     local spiritGuid = spiritChoices[spirit].guid
     if isSpiritPickable({guid = spiritGuid}) then
+        -- Return the unchosen spirits to the pool
```

Comment:
this is a tough one to think about. the original idea of having them still taken up was because this was based on the concept of drafting power cards. where you draw 4 and keep 1. but the others are discarded and players can't get them. but now with the concept of rerolling this is a weird one, not too sure which way i'd want to go

Reply:
If all players choose Gain a Spirit before any select one, it'll still be more around the concept of drafting cards.

Without this change, a "Low Complexity Spirits Only" game gives the first two players both 4 selections... and the 3rd gets 1.  Without it, the third can reroll after either/both of the first two have chosen to get a chance at the spirits in the pool.